docs/test/replication-controller-list.md

```diff
@@ -0,0 +1,39 @@
+# Replication controller list
+
+## Prerequisites
+Cluster with few deployed applications, where: 
+
+* at least one is valid
+* at least one is invalid (for example with wrong image) 
+
+## Expected result
+Page displays correctly, especially:
+
+* cards are displayed correctly even when page is scaled down/up
+* application details are valid and matching data from `kubectl` command
+* logs menu display correct pods information
+* all application actions are available
+* invalid applications have warning symbols and error messages
+
+## Specific factors
+
+| Factor                               | 1                                                | 2                                             | 3                                        | 4                                       | 5                                                                 | 6                                                                   | Comment |
+|--------------------------------------|--------------------------------------------------|-----------------------------------------------|------------------------------------------|-----------------------------------------|-------------------------------------------------------------------|---------------------------------------------------------------------|---------|
+| Replication controller count on page | 1 replication controller on wide page            | 1 replication controller on narrow page       | 3 replication controller on on wide page | 3 replication controller on narrow page | Any other positive number of replication controllers on wide page | Any other positive number of replication controllers on narrow page |         |
+| Replication controller status        | Valid                                            | Invalid (for example with not-existing image) |                                          |                                         |                                                                   |                                                                     |         |
+| Replication controller data          | Image string length (short or too long for card) | Age                                           | Logs (with links)                        | Internal and external endpoints         | Number of running and desired pods                                | Labels                                                              |         |
```

Comment:
RC fields | normal | max-lenght | high pod count
Label  | normal | long-text (253 prefix + 63 label) | 1 | odd | even | short-text & long-text mixed | 20 -30       Comment: labels increase the height of the card

- I split normal fields and label, because label has much more complexity. 
- I droped age, because I did not see enough complexity
- Pod count has some dependency to logs combo box. e.g. it seems that only the first 10 logs are shown even if the pod count is higher. A bug? Because of this dependency, I kept the field